### DISCUSSION TITLE
What is missing for V5?

### DISCUSSION THREAD

**Discussion by qbinary** [sanni/cartreader#371] [General Chat]
I plan to build the V5 but came across the warning that the shopping list will eventually change in the future. 
What are the expected changes? Are they just the missing 3D case model or core components?
In case that just the 3D case files are missing, I would design my own.

Amazing project btw. I really appreciate the effort :D

**Comment by sanni** [comment]
I have not received the V5 main PCB yet so I don't know if everything works out as planned.

**Reply by qbinary** [reply]
Ah ok, thanks for the response :D

**Comment by sanni** [comment]
Short update: I have received the PCB, everything is working but I do not like the location of the switches as they are awkward to operate with the USB cable being in the way all the time.

I'm going to test the PCB for a few days to see if anything else could be made better and then do a new revision that places the switches somewhere else.

![hw5rev1 (1)](https://user-images.githubusercontent.com/19169494/171598283-f720a41f-de30-49fd-acc4-19bcfa711217.jpg)

![hw5rev1 (2)](https://user-images.githubusercontent.com/19169494/171598286-d65dfe65-3f0a-4696-990b-a359f2efdd28.jpg)

![IMG_1301](https://user-images.githubusercontent.com/19169494/171905086-2ced8288-ae45-486c-a921-9e04ef723222.jpg)

![IMG_1305](https://user-images.githubusercontent.com/19169494/171905090-ccba0e59-2192-43b7-98c8-dd6645c14b89.jpg)

![IMG_1306](https://user-images.githubusercontent.com/19169494/171905093-0d126693-2255-4ec1-bfbf-c65e254dfcf6.jpg)
